packages/api/src/controllers/stream.ts

```diff
@@ -1790,7 +1776,7 @@ app.patch(
     const stream = await db.stream.get(id);
 
     const exists = stream && !stream.deleted;
-    const hasAccess = stream?.userId === req.user.id || req.user.admin;
+    const hasAccess = hasAccessToResource(req, stream, true);
     if (!exists || !hasAccess) {
```

Comment:
You can also just use the full `req.checkResourceAccess` helper here. This was just a dummy example of when the `hasAccessToResource` was needed (but not where the logic actually needed a separate check).